src/main/resources/avro/genotypephenotypemethods.avdl

```diff
@@ -101,6 +150,11 @@ full feature/phenotype/evidence objects.
 */
 record SearchGenotypePhenotypeRequest {
 
+  /**
+  The `PhenotypeAssociationSet` to search.
+  */
+  string phenotypeAssociationSetId;
+
   union {null, string, ExternalIdentifierQuery, OntologyTermQuery,
```

Comment:
These need to be better specified and documented. For example, as @diekhans mentioned, does it make sense to have an external  ID query on `Evidence` when evidence doesn't have an external ID field? What are the strings matching on?